requirements.txt

```diff
@@ -1,7 +1,6 @@
-numpy~=1.24.1
-robotpy[apriltag,cscore]~=2023.4.2
-robotpy-ctre~=2023.1.0
-robotpy-navx~=2023.0.3
-robotpy-rev~=2023.1.3.2
-robotpy-photonvision~=2023.3.0
-robotpy-wpilib-utilities~=2023.1.0
+numpy~=1.26.0
+robotpy[apriltag,cscore]~=2024.0.0b3
+robotpy-ctre~=2024.0.0b1
```

Comment:
We should use the latest beta here. There aren't any guarantees older vendor beta libraries will work with newer roboRIO/WPILib betas.